masonite/packages.py

```diff
@@ -46,6 +46,16 @@ def append_web_routes(location):
     package_routes.close()
     print('\033[92mroutes/web.py File Appended!\033[0m')
 
+def append_file(from_location, to_location):
```

Comment:
I believe you should use a context manager. Not only is it the standard way to mess with files, I believe it is also cleaner.

```python
with open(from_location, "r") as from_fp, open(to_location, 'a') as to_fp:
    to_fp.write(from_fp.read())
```

Reply:
actually didn't know you can use context managers like that